src/web_backups/main/ui/createConnection/CreateConnection.java

```diff
@@ -0,0 +1,30 @@
+package web_backups.main.ui.createConnection;
```

Comment:
This should be moved to a different package as it is not a UI functionality but making a connection to site specified in config. file. Please move this into global library.